File: Logic_HandsOn/Java/Problems on bit,file handling,String etc/Program179.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Write Java program which accept N numbers from user and display all such elements which are divisible by 3 and 5.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class ArrayX
{
	public int iSize,Brr[];

	public ArrayX(int iSize)
	{
		Brr = new int[iSize];
	}

	public void Accept()
	{
		int iCnt=0;
		Scanner sObj = new Scanner(System.in);
		System.out.println("Enter elements in Array: ");
		for(iCnt=0;iCnt<Brr.length;iCnt++)
		{
			Brr[iCnt] = sObj.nextInt();
		}
	}
}

class ArrayDemo extends ArrayX
{

	public ArrayDemo(int iValue)
	{
		super(iValue);
	}

	public void Display()
	{
		int iCnt=0;
		for(iCnt=0;iCnt<Brr.length;iCnt++)
		{
			if(Brr[iCnt]%5==0 && Brr[iCnt]%3==0)
			{
				System.out.print(Brr[iCnt]+" ");
			}
		}

		System.out.println();
	}		
}

class Program4

{
	public static void main(String arg[])
	{
		int iCnt=0;

		Scanner sObj = new Scanner(System.in);
		System.out.println("Enter number of elements: ");
		int iSize = sObj.nextInt();

		ArrayDemo aObj = new ArrayDemo(iSize);
		aObj.Accept();
		aObj.Display();
	}
}

/*
OUTPUT:

Enter number of elements: 
6
Enter elements in Array: 
85
66
3
15
93
88
15 

*/

File: Logic_HandsOn/C++/Problems on bit,File Handing,Pattern etc/program179.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Write Java program which accept N numbers from user and display all such elements which are divisible by 3 and 5.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

class ArrayX
{
	public:
		int iSize,*Arr;

		ArrayX(int iSize)
		{
			this->iSize=iSize;
			Arr = new int[iSize];	
		}

		void Accept()
		{
			cout<<"Enter elements: \n";
			for(int iCnt=0;iCnt<6;iCnt++)
			{
				cin>>Arr[iCnt];
			}
		}

		~ArrayX()
		{
			delete []Arr;
		}
};

class ArrayDemo:public ArrayX
{
	public:
		ArrayDemo(int x):ArrayX(x){}

		void Display()
		{
			cout<<"Output: ";
			int iCnt=0,iEven=0,iOdd=0;
			for(iCnt=0;iCnt<iSize;iCnt++,Arr++)
			{
				if(*Arr%3==0 && *Arr%5==0)
				{
					cout<<*Arr<<" ";
				}
			}

			cout<<"\n";
		}
};

int main()
{
	int iValue=0;
	cout<<"Enter the size of Array: ";
	cin>>iValue;

	ArrayDemo *aObj = new ArrayDemo(iValue);
	aObj->Accept();
	aObj->Display();
	return 0;
}

/*
OUTPUT:

Enter the size of Array: 6
Enter elements: 
85
66
3
80
93
88
Output:15

*/

